src/test/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/OrganizationMapperTest.java

```diff
@@ -11,26 +11,18 @@
 class OrganizationMapperTest {
 
     @Test
-    void testMapPerformingOrganization() {
+    void testMapToPerformingOrganization() {
         final Message message = new Message(new ArrayList<>());
 
         assertThat(new OrganizationMapper().mapToPerformingOrganization(message))
```

Comment:
When I run this locally, I get a compiler error about the lack of a `UUIDGenerator` parameter
```
error: constructor OrganizationMapper in class OrganizationMapper cannot be applied to given types;
        assertThat(new OrganizationMapper().mapToPerformingOrganization(message))
                   ^
  required: UUIDGenerator
  found: no arguments
  reason: actual and formal argument lists differ in length
```

Reply:
Sorry I mentioned this on Teams but not on the PR description. I'm still working on those unit tests but didn't want that blocking some reviews